### PR TITLE
Upgrade Gatsby to version 5

### PR DESCRIPTION
**Pull request by lubo**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16
+18
```
Comment (mitas1) [inline .nvmrc:1]: So Cloudflare has started to support this version?

Comment (mitas1) [reply .nvmrc:1]: OK, I've missed the commit description.

**Review by mitas1 — APPROVED** [review]
LGTM

File: .nvmrc (hunk 1 above)
Comment (lubo) [reply .nvmrc:1]: Explained in f53764e621d41316d2b4b45de0be7959b2b990b8.

**Pull request merged**